{{ cookiecutter.project_name }}/runtime.txt

```diff
@@ -1 +1 @@
-python-3.5.1
```

Comment:
Any reason we shouldn't be using python 3.5? Also it looks like the latest version of 3.4 is https://www.python.org/download/releases/3.4.4/